### PR TITLE
IG Brief - Attempt to fix locality issues.

### PR DESCRIPTION
**Pull request by EM-Creations**
**When merged this pull request will:**
- There were lots of locality issues here where it would only work in the editor, I think it should be OK now but please feel free to double check?

### CODE REVIEW

File: modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf [hunk 1]
```diff
@@ -5,6 +5,6 @@ BlueBoard removeAction GVAR(canceledBlu);
 
 [-2, {hint "Blufor has signaled they are ready to begin!";}] call CBA_fnc_globalExecute;
 
-GVAR(bluforIsReady) = True;
+SETMVAR(bluforIsReady, True);
 BlueBoard hideObjectGlobal true;
```
Comment (PiZZAD0X) [inline modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf:9]: doesn't hideObjectGlobal need to be executed on the server? From the server using that command to propagate to all clients?

File: modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf [hunk 2]
```diff
@@ -5,6 +5,6 @@ BlueBoard removeAction GVAR(canceledBlu);
 
 [-2, {hint "Blufor has signaled they are ready to begin!";}] call CBA_fnc_globalExecute;
```
Comment (PiZZAD0X) [inline modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf:6]: No reason to use CBA_fnc_globalExecute anymore, either remoteExec or a cba event

File: modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf [hunk 3]
```diff
@@ -5,6 +5,6 @@ BlueBoard removeAction GVAR(canceledBlu);
...
 
-GVAR(bluforIsReady) = True;
+SETMVAR(bluforIsReady, True);
```
Comment (PiZZAD0X) [inline modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf:8]: are these values being sent over the network somewhere?

Comment (EM-Creations) [reply modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf]: Think it's sorted now?

File: modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf (hunk 2 above)
Comment (EM-Creations) [reply modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf]: Done.

File: modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf (hunk 1 above)
Comment (EM-Creations) [reply modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf]: Done.

**Review by PiZZAD0X — APPROVED** [review]

**Pull request merged**